src/plugins/intel_cpu/src/nodes/space_to_batch.cpp

```diff
@@ -225,6 +225,9 @@ void SpaceToBatch::SpaceToBatchKernel() {
                                     const int64_t tmpOc = i5 * blockShape[1] + addTmpOc;
                                     const size_t srcIdx5 = srcIdx4 + it * outSpatialStep * blockSize + (tmpOc - it * blockSize);
                                     const size_t dstIdx5 = dstIdx4 + i5;
+                                    if (srcIdx5 >= srcLen || dstIdx5 >= dstLen) {
+                                        continue;
```

Comment:
Seems like we can use even `break` here because `srcIdx5` and `dstIdx5` are increased with `i5`. So if we get `srcIdx5 >= srcLen` or `dstIdx5 >= dstLen`, the next their values will be more than the corresponding lengths as well.

Reply:
There are also subtraction operations, thus it's safer to leave `continue` here.

Reply:
We subtract `it * blockSize` which is constant here (for nested loop). So I believe it's safely. But we can leave so asl well. No problems :smile: